codegen/core/src/main/scala/io/kalix/codegen/DescriptorSet2.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) Lightbend Inc. 2021
+ *
+ */
+
+package io.kalix.codegen
+
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.io.IOException
+import java.io.InputStream
+import java.util.logging.Level
+import java.util.logging.Logger
+
+import scala.jdk.CollectionConverters._
+import scala.util.Failure
+import scala.util.Success
+import scala.util.Using
+
+import com.google.protobuf.ExtensionRegistry
+import com.google.protobuf.DescriptorProtos
+import com.google.protobuf.Descriptors
+
+/**
+ * Provides conveniences for reading and parsing Protobuf descriptor sets
+ */
+object DescriptorSet2 {
```

Comment:
Any reason to keep the other one around?

Reply:
No, it was just a test, I'm cleaning this now.